src/rules.js

```diff
@@ -2687,5 +2687,72 @@ module.exports = [
         "arguments": [{
             "a": "auto"
         }]
+    },
+    /**
+    ==================================================================
+    SVG
+    ==================================================================
+    */
+    {
+        "type": "pattern",
+        "name": "fill",
+        "matcher": "Fill",
+        "allowParamToValue": true,
+        "styles": {
+            "fill": "$0"
+        },
+        "arguments": [colors]
+    },
+    {
+        "type": "pattern",
+        "name": "stroke",
+        "matcher": "Stk",
+        "allowParamToValue": true,
+        "styles": {
+            "stroke": "$0"
+        },
+        "arguments": [colors]
+    },
+    {
+        "type": "pattern",
+        "name": "stroke",
+        "matcher": "Stkw",
+        "allowParamToValue": true,
+        "styles": {
+            "stroke": "$0"
+        },
+        "arguments": [{
+            "i": "inherit"
+        }]
+    },
+    {
+        "type": "pattern",
+        "name": "stroke-linecap",
+        "matcher": "Stklc",
+        "allowParamToValue": true,
```

Comment:
I think this should be `false`: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-linecap